faust/sensors/monitor.py

```diff
@@ -325,6 +364,20 @@ def on_send_completed(self, producer: ProducerT, state: Any) -> None:
     def count(self, metric_name: str, count: int = 1) -> None:
         self.metric_counts[metric_name] += count
 
+    def on_tp_commit(self, tp_offsets: Mapping[TP, int]) -> None:
+        self.tp_committed_offsets.update({
+            tp: offset
+            for tp, offset in tp_offsets.items()
+            if offset is not None
+        })
+
+    def track_tp_end_offsets(self, tp_end_offsets: Mapping[TP, int]) -> None:
+        self.tp_end_offsets.update({
```

Comment:
Do we have to remove the None values here?
Can't we just do `self.tp_end_offsets.update(tp_end_offsets)` ?

Reply:
Could do that, and then handle it in statsd. But I am not sure if the monitor should ever display None values?

Reply:
That will also make the offsets Mapping have type Mapping[TP, Optional[int]] which seems a bit weird to me.

Reply:
Actually now that I think of it, the values should never be None.